### ISSUE TITLE
Update to actix-web 4.0

### ISSUE THREAD

**Issue by izolyomi** [paperclip-rs/paperclip#292]
Hey, I really appreciate the effort put into this convenient crate.

As Actix-web 4.0 is coming close after Tokio 1.0 has arrived, are there any plans to update Paperclip to these new dependency versions?

**Comment by frol** [comment]
FYI, I [went ahead and checked how much work it would take to migrate to actix-web 4 beta](https://github.com/wafflespeanut/paperclip/compare/master...frol:actix-web-4-beta), and while it is cruel way without proper support for both actix3 and actix4, I think it serves the purpose and someone at least can start from here. My `actix-web-4-beta` branch compiles fine with `actix` feature, and it compiles fine as a part of my project that uses `paperclip` as a dependency and I confirm a successful migration to Tokio 1.x

**Comment by tiagolobocastro** [comment]
@frol if you have the time a PR would be appreciated :)

**Comment by frol** [comment]
Sorry, I don't have the capacity to submit a proper PR with backward compatibility.

Here is another update to the latest beta.3: https://github.com/wafflespeanut/paperclip/compare/master...frol:actix-web-4-beta.3

**Comment by tiagolobocastro** [comment]
Thanks @frol :+1:  I'll use that as a base but I'll wait for now since actix 4 is still in beta.

**Comment by Uzaaft** [comment]
Any updates regarding support for actix 4?

**Comment by tiagolobocastro** [comment]
Please see: https://github.com/paperclip-rs/paperclip/pull/314

**Comment by TobiasDeBruijn** [comment]
With #314 closed, can a new version of Paperclip be published to crates.io? Thanks!

**Comment by mojerro** [comment]
hope to support actix4, and now we can't use it with actix4 😺

**Comment by mojerro** [comment]
and the document says that supports actix4 which makes me trouble

```
paperclip = { version = "0.6.1", features = ["actix4"] }

```

**Comment by TobiasDeBruijn** [comment]
Paperclip with actix 4 support is not yet available on crates.io. Use git as a source or use patching to be able to use it for now 😄

**Comment by mojerro** [comment]
I've seen, and I'm now using it with code
```
[dependencies.paperclip]
version = "0.6"
git = "https://github.com/paperclip-rs/paperclip"
branch = "master"
features = ["actix4", "v3", "swagger-ui", "actix-files"]
```

**Comment by tiagolobocastro** [comment]
Resolved by #314
